solutions/cf-1365D.mdx

```diff
@@ -40,97 +40,103 @@ typedef long long ll;
 const int mxN = 55;
 char grid[mxN][mxN];
 bool visited[mxN][mxN];
+int rowMovement[4] = {0, 1, 0, -1};
```

Comment:
```suggestion
int rowMovement[4]{0, 1, 0, -1};
```

Reply:
https://www.learncpp.com/cpp-tutorial/arrays-part-ii/

Reply:
wait doing this compiler errors

Reply:
i don't really do c++ that much so i don't really understand much